config/types/schema.go

```diff
@@ -0,0 +1,179 @@
+package types
+
+// generated by "schematyper --package=types schema/ignition.json -o config/types/schema.go --root-type=Config" -- DO NOT EDIT
+
+type Config struct {
+	Ignition IgnitionVersion `json:"ignition"`
```

Comment:
This type isn't correct. The Ignition structure contains more than just the version.